entrypoint.sh

```diff
@@ -28,7 +28,7 @@ then
 fi
 
 case "$1" in
-	  -- | openerp-server | /mnt/odoo-source/odoo-bin)
+	  -- | odoo-server | /mnt/odoo-source/odoo-bin)
```

Comment:
odoo-server doesn't make sense here. Probably we can make it as in official docker:
https://github.com/odoo/docker/blob/master/11.0/entrypoint.sh#L27

    -- | odoo)